scripts/build/release.sh

```diff
@@ -154,7 +154,7 @@ echo "new version:     $NEW_VERSION"
 #
 # Edit files that depend on the current version (which just changed)
 #
-sed "s/$currentVersion/$NEW_VERSION/" package.json        > /tmp/package.json
+sed "s/\"version\": $currentVersion/\"version\": $NEW_VERSION/" package.json        > /tmp/package.json
```

Comment:
Please, pay special attention to this change. I'm not an expert in sed and I haven't tested it directly (difficult to test the release.sh script, as we only use it for producing release). Do you think it will work?